watch-faces/complication/totp_lfs_face.c

```diff
@@ -242,7 +245,6 @@ static void totp_face_set_record(totp_lfs_state_t *totp_state, int i) {
 }
 
 void totp_lfs_face_activate(void *context) {
-    memset(context, 0, sizeof(totp_lfs_state_t));
```

Comment:
Doing the memset here is intentional, as it means that it will reload if LFS is updated.

At the moment, this is only valid in the emulator or if you have a hacked up watch, but in the future... irDA?

If you remove it, even so, I think you probably want to reset back to the first thing in the rotation.

Is this change related to the movement_pin_service?

Reply:
@wryun  Ok, I understand the motivation now.

The issue with `memset` is that it throws away everything, including the `face_index` I now store in the state (used for pin redirect).
So I guess the solution is to explicitly reset the parts of the state that are meant to be reset.

I pushed this change, let me know if it would work

Reply:
Ah, I think I forgot/misunderstood what the original code was doing anyway, sorry! Doesn't seem to have anything to do with LFS load, just the state reset. This seems fine. Sorry, don't have time to test it out at the moment.